tests/integration/test_keys.py

```diff
@@ -7,7 +7,7 @@ def test_types():
     r.set('mystr', 'test')
     r.incr('myint')
     r.sadd('myset', 'test')
-    r.zadd('myzset', 0, 'test')
+    r.zadd('myzset', {'test': 0})
```

Comment:
New syntax for Python's redis client, more at https://github.com/andymccurdy/redis-py#mset-msetnx-and-zadd